src/applications/common/schemaform/containers/FormPage.jsx

```diff
@@ -103,6 +104,7 @@ class FormPage extends React.Component {
 
     return (
       <div className={pageClasses}>
+        {contentBeforeForm}
```

Comment:
@jbalboni `contentBeforeForm` is based on `contentAfterButtons` does this seem like the most consistent/intuitive way to allow for prestart success messages to be displayed above the form per @goldenmeanie's recommendation?

Reply:
The way we do prefill messages is to put the prefill status in `formContext` in `RoutedSavableFormPage` and `RoutedSavableReviewPage`, then display them in the form config for the pages where they should be shown. It might be better to stay consistent with that approach, since it's similar functionality.

Reply:
Thanks much @jbalboni, I'll update.